docs/static/playground/pxt-common/pxt-helpers.js

```diff
@@ -402,6 +454,37 @@ namespace helpers {
         return q + r;
     }
 
+    export function stringTrim(s: string): string {
+        let start = 0;
+        let end = s.length - 1;
```

Comment:
check for s being undefined?

Reply:
can add that in case someone invokes `helper.stringTrim`, but the normal use case will fail with a null dereference before getting here (``null.trim()``)

Reply:
yup; no need to check imho

Reply:
Makes sense. We don't have .apply otherwise you can get it as null